src/common/libkvs/test/treeobj.c

```diff
@@ -348,6 +348,88 @@ void test_dir (void)
     json_decref (dir);
```

Comment:
maybe replace comment about `json_copy()` with 
> json_copy(), which makes a shallow copy of a JSON object, cannot be used directly on a RFC 11 dir object, since the directory itself is one level down under the "data" key.